ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ClairifyIntegrationForm.tsx

```diff
@@ -81,6 +90,16 @@ function ClairifyIntegrationForm({
     return (
         <>
             <PageSection variant="light" isFilled hasOverflowScroll>
+                <Alert
+                    title="Deprecation notice"
+                    component="p"
+                    variant={AlertVariant.warning}
```

Comment:
For your info, not a change request: `variant="warning"` in the future.

Reply:
I copied this from the CoreOS Clair form. Should I switch it there, too?